ocs_ci/ocs/resources/pod.py

```diff
@@ -831,6 +831,82 @@ def get_ocs_provider_server_pod(label=constants.PROVIDER_SERVER_LABEL, namespace
     return Pod(**ocs_provider_server[0])
 
 
+def get_lvm_vg_manager_pod(
+    label=constants.LVMO_POD_LABEL["411"]["vg-manager_label"],
```

Comment:
I think it is better not to have default values set with a specific version

Reply:
i will change it once Shay's PR will be mergred